test/Project.toml

```diff
@@ -3,7 +3,10 @@ BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 CUDAKernels = "72cfdca4-0801-4ab0-bf6a-d52aa10adc57"
+CloudMicrophysics = "6a9e3e04-43cd-43ba-94b9-e8782df3c71b"
```

Comment:
We don't want to add this dependency to the tests, because it will mess up with the CI setup. It's ok to leave it for now, so that things work for us locally. But we should remove it before merging